src/modules/rc_update/rc_update.cpp

```diff
@@ -340,8 +345,23 @@ RCUpdate::Run()
 
 	/* read low-level values from FMU or IO RC inputs (PPM, Spektrum, S.Bus) */
 	input_rc_s rc_input;
+	bool input_rc_copy_ok = false;
+	if (_input_rc_subs[0].updated()) {
+		input_rc_copy_ok = _input_rc_subs[0].copy(&rc_input);
+	}
+
+	/* if the first instance was not updated and other instances are allowed,
+	 * find the one that triggered the callback */
+	if (!input_rc_copy_ok && _param_com_rc_multiple_inputs.get() > 0) {
+		for (int i = 1; i < MAX_INPUT_RC; i++) {
+			if (_input_rc_subs[i].updated()) {
```

Comment:
Is it ok to check if it is updated or should I check which rc_input.timestamp is the last ?